### PR TITLE
Update Peering SDK to 2020-04-01 API version

### PR DESCRIPTION
**Pull request by psanapathi**

### CODE REVIEW

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hello @psanapathi 
- Please add a link to the openapi spec review PR here
- `peering_resource-manager.txt` is in old format, please [update your build tools](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools) and make sure the SDK is generated by running `src/generate.ps1`
- Update api tag in `AzSdk.RP.props`, release notes in csproj
- Update versions in csproj and `assemblyinfo.cs` if you are going to release

Thanks

**Comment by psanapathi** [comment]
> Hello @psanapathi
> 
> * Please add a link to the openapi spec review PR here
> * `peering_resource-manager.txt` is in old format, please [update your build tools](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools) and make sure the SDK is generated by running `src/generate.ps1`
> * Update api tag in `AzSdk.RP.props`, release notes in csproj
> * Update versions in csproj and `assemblyinfo.cs` if you are going to release
> 
> Thanks
 
- API spec PR: https://github.com/Azure/azure-rest-api-specs/pull/9361
- Metadata is updated now
- API tag, version, and release notes are updated now

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj [hunk 1]
```diff
@@ -7,10 +7,12 @@
 		<PackageId>Microsoft.Azure.Management.Peering</PackageId>
 		<Description>Microsoft Azure Management Peering library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Peering</AssemblyName>
-		<Version>2.0.1-preview</Version>
+		<Version>1.0.0</Version>
```
Comment (isra-fel) [inline sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj:10]:
```suggestion
		<Version>2.1.0</Version>
```
We do not decrease versions according to our versioning guide: https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md
Also this is feature addition to 2.0.1-preview.

Comment (isra-fel) [reply sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj:10]: Please update the versions in `AssemblyInfo.cs` as well

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj (hunk 1 above)
Comment (psanapathi) [reply sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj]: Updated.

**Review by isra-fel — APPROVED** [review]
LGTM!

**Pull request merged**